src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -343,7 +343,7 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     // Filter disabled plugins:
     const activePlugins = bestOfPlugins(accountPlugins, accountReferral, undefined)
     plugins = plugins.filter(plugin => !activePlugins.disabled[plugin.pluginId])
-    plugins = plugins.filter(plugin => !disablePlugins[plugin.pluginId])
+    plugins = plugins.filter(plugin => disablePlugins[plugin.pluginId] !== true)
```

Comment:
This didn't consider my comment that you need to change this to `[plugin.id]`.

`[plugin.pluginId]` is shared across multiple payment methods so it doesn't allow us to disable with fine granularity.